CONTRIBUTORS.md

```diff
@@ -31,3 +31,4 @@
 * Phil Osip
 * Sara Chaparro Diaz
 * Seth Vincent
+* Snyk bot
```

Comment:
Snyk bot really shouldn't be a contributor

Reply:
Yeah but our script to upgrade the contributors list always adds it, I assume because of the commit history. And it will be annoying to keep having to ignore this change

Reply:
That sounds like a script update to me, if so.   Nevertheless it's not *directly* germaine to this PR.